korge/src/commonMain/kotlin/com/soywiz/korge/html/Html.kt

```diff
@@ -71,7 +71,7 @@ object Html {
 
 		object Identity : MetricsProvider {
 			override fun getBounds(text: String, format: Format, out: Rectangle) {
-                out.setTo(0, 0, text.length, 1)
+                out.setTo(0.0, 0.0, text.length, 1.0)
```

Comment:
This was using the integral overload. `text.length` was an integer too.  This would end using the inline `Number` overload that should be deprecated temporarily since we had performance issues with that kind of functions on Kotlin/Native.

So if you plan to change it to use the Double overload you should do:

```
out.setTo(0.0, 0.0, text.length.toDouble(), 1.0)
```

I will mark the `Rectangle.setTo` with mixed numbers deprecated for now to avoid confusions, until this issue is resolved: https://youtrack.jetbrains.com/issue/KT-39136